addons/mail/static/src/core/common/composer.xml

```diff
@@ -35,8 +35,8 @@
                 <div class="d-flex o-mail-Composer-bg flex-grow-1 border"
                     t-att-class="{
                         'o-mail-Composer-compactContainer m-1 shadow-sm border-secondary': compact and !props.composer.message,
-                        'o-mobile rounded-3': isMobileOS,
-                        'rounded-3' : normal,
+                        'o-ios-mobile': isIosApp,
```

Comment:
I would name classname `o-iosPwa` for clarity (`ios-mobile`, this is already a mobile OS, but it's unclear whether this covers the app or in browser, both of which shouldn't be taken into account so name clarifies it)